COPY/etc/issue.template

```diff
@@ -1,4 +1,4 @@
-Welcome to the \P Virtual Appliance.
+Welcome to the Virtual Appliance.
```

Comment:
Hmm, I prefer to have the product name like we had before, as Virtual Appliance is too generic.